oak_functions/client/java/tests/com/google/oak/functions/client/ServerConfigurationVerifierTest.java

```diff
@@ -20,8 +20,8 @@
 import oak.functions.abi.ConfigurationInfo;
 import org.junit.Assert;
 import org.junit.Test;
-import org.junit.runner.RunWith;
```

Comment:
I think this has nothing to do with this change, it just happens to be called similarly.

Reply:
Hey, sorry for late response, got a little busy with work, I have made changes accordingly.